src/main/java/org/datacleaner/extension/elasticsearch/ElasticSearchIndexAnalyzer.java

```diff
@@ -84,6 +84,9 @@
     @Configured(required = false)
     boolean automaticDateDetection = false;
 
+    @Configured(required = false)
+    boolean keepStringFieldsNotAnalyzedAlso = true;
```

Comment:
"also" seems like a bad thing to slip into the component interface. I'm not sure as a casual user what the also refers to, is it in addition to something else?